src/Templates/Microsoft.CodeAnalysis.AnalyzerTemplate/.template.config/template.json

```
@@ -0,0 +1,31 @@
+
+
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "???",
```

What should the author be here?

---

@ryzngard Other dotnet templates ([console](https://github.com/dotnet/templating/blob/master/template_feed/Microsoft.DotNet.Common.ProjectTemplates.5.0/content/ConsoleApplication-CSharp/.template.config/template.json), [classlib](https://github.com/dotnet/templating/blob/master/template_feed/Microsoft.DotNet.Common.ProjectTemplates.5.0/content/ClassLibrary-CSharp/.template.config/template.json), and [asp templates](https://github.com/dotnet/aspnetcore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/.template.config/template.json)) have author as "Microsoft"

So I think this should be the same.

```suggestion
    "author": "Microsoft",
```

---

I think it should me `Microsoft` as well